src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  max-width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+  padding-inline: 16px;
+  transition-duration: all 1s;
```

Comment:
Using `all` in transition-duration can lead to performance issues as it will animate all properties. Be specific about which properties you want to animate. For example, if you want to animate color, use `transition: color 1s;` instead.